pyanaconda/modules/payloads/source/url/url.py

```diff
@@ -67,6 +68,46 @@ def for_publication(self):
         """Get the interface used to publish this source."""
         return URLSourceInterface(self)
 
+    def process_kickstart(self, data):
+        """Process the kickstart data."""
+        repo_data = RepoConfigurationData()
+        repo_data.name = self._url_source_name
+
+        if data.url.url:
+            repo_data.url = data.url.url
+            repo_data.type = URL_TYPE_BASEURL
+        elif data.url.mirrorlist:
+            repo_data.url = data.url.mirrorlist
+            repo_data.type = URL_TYPE_MIRRORLIST
+        elif data.url.metalink:
+            repo_data.url = data.url.metalink
+            repo_data.type = URL_TYPE_METALINK
+
+        repo_data.proxy = data.url.proxy
+        repo_data.ssl_verification_enabled = not data.url.noverifyssl
+        repo_data.ssl_configuration.ca_cert_path = data.url.sslcacert
+        repo_data.ssl_configuration.client_cert_path = data.url.sslclientcert
+        repo_data.ssl_configuration.client_key_path = data.url.sslclientkey
```

Comment:
Are you sure this cannot be None?

Reply:
I don't think it will be. In general there should always be `False`/`True`. I would leave it like this and we can fix it later if there will be an error later.

Reply:
Fixed I thought we are talking just about the first line.